src/platform/graphics/webgl/webgl-shader.js

```diff
@@ -227,6 +227,11 @@ class WebglShader {
             const message = "Failed to link shader program. Error: " + gl.getProgramInfoLog(glProgram);
 
             // #if _DEBUG
+
+            // log translated shaders
+            definition.translatedFrag = gl.getExtension('WEBGL_debug_shaders').getTranslatedShaderSource(this.glFragmentShader);
+            definition.translatedVert = gl.getExtension('WEBGL_debug_shaders').getTranslatedShaderSource(this.glVertexShader);
```

Comment:
This _could_ return `null` right?

Reply:
it could, and that is fine, the error console log will have undefined/null for the field.

Reply:
ah sorry, I missed what you're saying, I'll fix this :)

Reply:
No, `getTranslatedShaderSource` will through an exception, surely (called by `null`).